plugins/discourse-lazy-videos/assets/javascripts/discourse/components/lazy-iframe.js

```diff
@@ -4,7 +4,7 @@ export default class LazyVideo extends Component {
   get iframeSrc() {
     switch (this.args.providerName) {
       case "youtube":
-        return `https://www.youtube.com/embed/${this.args.videoId}?autoplay=1`;
+        return `https://www.youtube.com/embed/${this.args.videoId}?autoplay=1&start=${this.args.startTime}`;
```

Comment:
Not a blocker by any means, but like with vimeo below, I think it may be cleaner to omit the `start` query param if it's 0.

Reply:
Yes no strong opinions on this, will change